src/SDKs/SqlManagement/Management.Sql/Properties/AssemblyInfo.cs

```diff
@@ -22,7 +22,7 @@
 [assembly: AssemblyTitle("Microsoft Azure SQL Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure SQL.")]
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.4.0.0")]
+[assembly: AssemblyFileVersion("1.4.0.1")]
```

Comment:
Is there any product code change, or just test code? If test only, then we don't need to publish so I don't think this needs to be bumped